app/views/layouts/application.html.erb

```diff
@@ -8,9 +8,12 @@
 
```

Comment:
![image](https://github.com/stephenkati/blog/assets/93039359/073ef2ad-0c60-45d4-896b-0b1a79a40cc4)

- [ ] It would be nice if you fix the highlighted links in the image above because these links are broken right now 😨 Kindly fix them or remove them to make your readme descriptive and professional 👍